modules/renter/contractor/formcontract.go

```diff
@@ -363,6 +367,7 @@ func (c *Contractor) formContracts(a modules.Allowance) error {
 			// negotiate with because they'll probably be more expensive than
 			// we can afford.
 			c.log.Println("WARN: failed to negotiate contract:", h.NetAddress, err)
```

Comment:
A renter in the wild was reported to have a log of 1.5GB, too much for non-dev machines. Target should be less than 1mb per day for all of `siad` on the logs, ideally more like 10k per day max for all of `siad`.

Dev machines should try to log as little as possible, but up to 1GB per day for all of `siad` is okay. If the information can help us improve the code, the disk cost is not really a concern.